io.github.bcpierce00.unison.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Exec=unison-gui
+Name=Unison
+GenericName=File Synchronizer
+Comment=Multi-master File synchronization tool
+Terminal=false
+Icon=io.github.bcpierce00.unison.svg
```

Comment:
this is always the icon name and NEVER the icon **FILE NAME*. ie, no extensions.